src/test/java/com/blackducksoftware/integration/hub/alert/processor/PolicyOverrideProcessorTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2017 Black Duck Software Inc.
+ * http://www.blackducksoftware.com/
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Black Duck Software ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Black Duck Software.
+ */
+package com.blackducksoftware.integration.hub.alert.processor;
+
+import static org.junit.Assert.assertTrue;
+
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.blackducksoftware.integration.hub.dataservice.model.ProjectVersionModel;
+import com.blackducksoftware.integration.hub.dataservice.notification.model.PolicyOverrideContentItem;
+import com.blackducksoftware.integration.hub.exception.HubIntegrationException;
+import com.blackducksoftware.integration.hub.model.view.ComponentVersionView;
+import com.blackducksoftware.integration.hub.model.view.PolicyRuleView;
+import com.blackducksoftware.integration.hub.notification.processor.MapProcessorCache;
+import com.blackducksoftware.integration.test.TestLogger;
+
+public class PolicyOverrideProcessorTest {
+
+    @Test
+    public void testProcessRemoveEvent() throws HubIntegrationException, URISyntaxException {
+        // Test will break if ListProcessorCache is used because NotificationEvents does not have an equals method defined
```

Comment:
Why not add an equals method?

Reply:
NotificationEvent is an object from another library. So we'd have to go through an entire release just to fix a test. We use the MapCache in alert so we should be using that in the test anyway, but maybe in the future we may want to add an equals method

Reply:
So then why is the comment necessary if we use a MapCache anyway?

Reply:
I wanted to worn people about a possible issue if it were changed. I think we should keep the comment, but maybe we can move it elsewhere